pytorch_toolkit/tests/run_model_templates_tests.py

```diff
@@ -22,18 +23,55 @@
 
 ENABLE_TRAIN_TESTS = True
 ENABLE_EXPORT_TESTS = True
+ENABLE_NNCF_TESTS = True
 
+def _is_verbose_flag_set():
```

Comment:
why not argparse? In your case there is not help

Reply:
please make a good args parser with ENABLE_TRAIN_TESTS.., WORK_DIR, .., and other stuff defined

Reply:
Ilya, I would like to do this change during the next week in the next PR.

Is it ok for you?

Reply:
ok